### PR TITLE
PR for Business Continuity ticket 8260

### PR DESCRIPTION
**Pull request by dilipkumarindla**
Draft PR Feature/fi0 8260 for Business Continuity ticket 8260

### CODE REVIEW

File: src/server/certificates/controller.test.js [hunk 1]
```diff
@@ -94,6 +95,125 @@ describe('#certificatesController', () => {
...
+    server = await createServer()
+    await server.initialize()
+  })
+
+  beforeEach(() => {
+    mockGetCertificate = jest.spyOn(Certificates, 'getList')
+    mockPutCertificate = jest.spyOn(Certificates, 'uploadCertificateDetails')
+    mockDeleteCertificate = jest.spyOn(Certificates, 'removeDocument')
+    mockGetCertificate.mockResolvedValue(result)
+    // mockPutCertificate.mockResolvedValue(result)
```
Comment (alexwbbr) [inline src/server/certificates/controller.test.js]: please remove these if they aren't being used

File: src/server/certificates/controller.test.js [hunk 2]
```diff
@@ -94,6 +95,125 @@ describe('#certificatesController', () => {
...
+  test('Should call the api certficates api successfully', async () => {
+    const { statusCode } = await server.inject({
+      method: 'GET',
+      headers: {
+        'x-api-key': apiHeaderKey
+      },
+      url: '/api/certificates'
+    })
+
+    expect(statusCode).toBe(200)
```
Comment (alexwbbr) [inline src/server/certificates/controller.test.js:139]: please add a check to see that a list of items are actually returned

File: src/server/certificates/controller.test.js (hunk 1 above)
Comment (dilipkumarindla) [reply src/server/certificates/controller.test.js]: Thanks, @alexwbbr - I missed removing the commented code. It was now updated.

File: src/server/certificates/controller.test.js (hunk 2 above)
Comment (dilipkumarindla) [reply src/server/certificates/controller.test.js:139]: Thanks, @alexwbbr - Updated now.

File: src/server/certificates/controller.test.js [hunk 3]
```diff
@@ -94,6 +95,135 @@ describe('#certificatesController', () => {
   })
 })
 
+describe('API calls for GET/PUT/DELETE', () => {
+  /** @type {Server} */
+  let server
+  let mockGetCertificate, mockPutCertificate, mockDeleteCertificate
+  const apiHeaderKey = config.get('apiAuth')
+  const result = {
```
Comment (alexwbbr) [inline src/server/certificates/controller.test.js]: it should be an array of objects, please check the ecert_certificates.json file

Comment (dilipkumarindla) [reply src/server/certificates/controller.test.js]: Resolved. Thanks @alexwbbr

**Review by alexwbbr — APPROVED** [review]

**Pull request merged**